### PR TITLE
Moving Tests Folder, Updating README.md and Gyro and accelerometer modules.

### PR DESCRIPTION
**Pull request by ron-popov**
In this pull request :  
* I moved the tests folder from the Modules folder (it was there only because it was a pain to run the tests otherwise) and added a script that runs a wanted test (it's kind of a pain to run tests manually).  
* Updated the README and added a section about the structure of the repo and running tests.  
* Gyro and accelerometer modules, they are pretty much the same as the magnetometer (since they both use the same hardware module), with the only difference in analyzing the data.

### CODE REVIEW

File: Modules/Accelerometer/accelerometer.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    def __get_y__(self):
+        """Measures y axis acceleration value"""
+        return str(self.__get_axes__()[1])
+
+    def __get_z__(self):
+        """Measures z axis acceleration value"""
+        return str(self.__get_axes__()[2])
+
+    def __get_axes__(self):
```
Comment (yotam180) [inline Modules/Accelerometer/accelerometer.py:30]: Just curious... Why the two underscores `__`?

Comment (yotam180) [reply Modules/Accelerometer/accelerometer.py:30]: Ah, is it supposed to be "protected"?

**Review by yotam180 — APPROVED** [review]
Seems alright.

File: Modules/Accelerometer/accelerometer.py (hunk 1 above)
Comment (ron-popov) [reply Modules/Accelerometer/accelerometer.py:30]: Yes

**Pull request merged**